state-chain/chains/src/dot.rs

```diff
@@ -331,7 +331,7 @@ impl Chain for Polkadot {
 	type ChainAsset = assets::dot::Asset;
 	type DepositFetchId = PolkadotChannelId;
 	type DepositChannelState = PolkadotChannelState;
-	type DepositDetails = ();
+	type DepositDetails = PolkadotExtrinsicIndex;
```

Comment:
i think there is no (external chain) block number on the `DepositFinalised`/`DepositBoosted`, so it would be nice to have that in the deposit detail (or on the event)

Reply:
yeah it's on the extrinsic but not the event... probably does make sense to have it on the event

Reply:
yes it would be nice to have it on the event (at least for polkadot, maybe in general)